lib/galaxy/dependencies/conda-requirements.txt

```diff
@@ -0,0 +1,63 @@
+# packages with C extensions
+# numpy should be installed before bx to enable extra features in bx
+numpy
+bx-python
+MarkupSafe
+PyYAML
+SQLAlchemy
+# Mercurial >= 3.5 changed the bundle format, which breaks hg push of TS repositories
+mercurial<3.5
+pycrypto
+
+# Install python_lzo if you want to support indexed access to lzo-compressed
+# locally cached maf files via bx-python
+#python_lzo
+
+# pure Python packages
+Paste
+PasteDeploy
+docutils
+#wchartype
+repoze.lru
+Routes
+WebOb
+#WebHelpers
+Mako
+pytz
+Babel
+#Beaker
+
+# Cheetah and dependencies
+Cheetah
+
+# BioBlend and dependencies
+bioblend
+boto
+requests
+
+# kombu and dependencies
+#kombu
+
+# sqlalchemy-migrate and dependencies
+#sqlalchemy-migrate
+decorator
+#Tempita
+sqlparse
+six
+#Parsley
+nose
+#SVGFig
+
+# Fabric and dependencies
+Fabric
+
+# We still pin these dependencies because of modifications to the upstream packages
+
+# Can't use Whoosh latest due to a bug but need to backport a bugfix from a
+# newer release.
+#
+# https://bitbucket.org/mchaput/whoosh/pull-requests/50/fix-lru-thread-safety/diff
+#Whoosh==2.4.1+gx1
```

Comment:
Here is what I would do - rename this to conda-environmnt.txt since it isn't a requirements.txt file and augment with a second ``conda-requirements.txt`` file that has the wheel locations.

```
conda create galaxy --file conda-environment.txt
source activate galaxy
pip install -r conda-requirements.txt
```